src/main/java/de/komoot/photon/json/JsonDumpConnector.java

```diff
@@ -0,0 +1,66 @@
+package de.komoot.photon.json;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import de.komoot.photon.Connector;
+import de.komoot.photon.ImportProgressMonitor;
+import de.komoot.photon.elasticsearch.Importer;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Import json dump data.
+ *
+ * @author holger
+ */
+@Slf4j
+public class JsonDumpConnector implements Connector {
+
+    private BufferedReader reader;
+    private Importer importer;
+
+    public JsonDumpConnector(String filename) {
+        try {
+            this.reader = new BufferedReader(new FileReader(filename));
+        } catch (FileNotFoundException e) {
+            log.error("Json dump file '%1' not found", filename);
+        }
+    }
+
+    public void setImporter(de.komoot.photon.Importer importer) {
```

Comment:
shouldn't it be provided on construction and then importer and reader could be final?